--- dirent.h.orig	2018-12-20 22:55:04.944569999 +0000
+++ dirent.h	2018-12-26 17:21:28.801786990 +0000
@@ -7,7 +7,6 @@
 #ifndef _dirent_h_
 #define _dirent_h_
 
-#include <sys/iosupport.h>
 #include <sys/types.h>
 #include <sys/syslimits.h>
 
@@ -31,6 +30,12 @@
 		char	d_name[NAME_MAX+1];
 	};
 
+/* Directory iterator for mantaining state between dir* calls */
+	typedef struct {
+		int device;
+		void *dirStruct;
+	} DIR_ITER;
+
 	typedef struct {
 		long int        position;
 		DIR_ITER*       dirData;
